conformance/pom.xml

```diff
@@ -177,17 +179,5 @@
                 </executions>
             </plugin>
         </plugins>
-        <resources>
```

Comment:
As long as filtering is done elsewhere...

Reply:
Yeah, again done in the master `pom.xml`.
Was repeated in more modules, Furthermore now needed for test resources recognition and the variables expansion in `..../test/resources/log4j.properties`